PhpStormStubsMap.php

```diff
@@ -5749,8 +5749,8 @@ final class PhpStormStubsMap
   'DB2_PASSTHRU' => 'ibm_db2/ibm_db2.php',
   'DB2_SCROLLABLE' => 'ibm_db2/ibm_db2.php',
   'DB2_XML' => 'ibm_db2/ibm_db2.php',
-  'DEBUG_BACKTRACE_IGNORE_ARGS' => 'Core/Core.php',
-  'DEBUG_BACKTRACE_PROVIDE_OBJECT' => 'Core/Core.php',
+  'DEBUG_BACKTRACE_IGNORE_ARGS' => 'Core/Core_d.php',
+  'DEBUG_BACKTRACE_PROVIDE_OBJECT' => 'Core/Core_d.php',
```

Comment:
Just a thought but maybe the stub map dumper should throw when multiple files declare the same symbol?

Reply:
@theofidry That is a good idea to implement! But I will add this check to the standard tests. Map tester should test map first of all!